CondTools/CTPPS/test/ppsTimingCalibrationWriter_cfg.py

```diff
@@ -9,23 +9,24 @@
     interval = cms.uint64(1)
 )
 
+from CondFormats.CTPPSReadoutObjects.PPSTimingDetEnum_cff import PPSTimingDetEnum
+
 # load calibrations from JSON file
 process.load('CondFormats.CTPPSReadoutObjects.ppsTimingCalibrationESSource_cfi')
-process.ppsTimingCalibrationESSource.calibrationFile = cms.FileInPath('RecoCTPPS/TotemRPLocal/data/timing_offsets_ufsd_2018.dec18.cal.json')
+process.ppsTimingCalibrationESSource.calibrationFile = cms.FileInPath('RecoCTPPS/TotemRPLocal/data/timing_calibration_diamond_2018_mar19.ex.json')
```

Comment:
@forthommel related to your comment https://github.com/cms-data/RecoCTPPS-TotemRPLocal/pull/3#issuecomment-476227584 : I understand this is just a test, but if that file is not merged the test will be broken and in practice useless. Will json files be routinely used?

Reply:
You are perfectly right. As it is now, this test is only used as a method to produce the sqlite payload prior to filling the database, therefore only indirectly in production. Same comment for the `ESProducer` itself. So no, theses JSON files will not be routinely used in production.